### PR TITLE
xtest: add command to get system time

### PR DESCRIPTION
**Pull request by weizhaojiang**
Command format: xtest --stats --time

Tested-by: Weizhao Jiang <weizhaoj@amazon.com>

### CODE REVIEW

**Review by etienne-lms — COMMENTED** [review]
`Acked-by: Etienne Carriere <etienne.carriere@foss.st.com>`

File: host/xtest/stats.c [hunk 1]
```diff
@@ -312,6 +313,36 @@ static int stat_loaded_ta(int argc, char *argv[])
...
+
+	UNUSED(argv);
+	if (argc != 1)
+		return usage();
+
+	open_sess(&ctx, &sess);
+	op.paramTypes = TEEC_PARAM_TYPES(TEEC_VALUE_OUTPUT,
+					 TEEC_VALUE_OUTPUT,
+					 TEEC_NONE, TEEC_NONE);
+	res = TEEC_InvokeCommand(&sess, STATS_CMD_SYSTEM_TIME, &op, &eo);
```
Comment (jforissier) [inline host/xtest/stats.c:332]: This command could be implemented in the "OS test" TA (`ta/os_test/os_test.c`).

Comment (weizhaojiang) [reply host/xtest/stats.c:332]: > This command could be implemented in the "OS test" TA (`ta/os_test/os_test.c`).

Thanks for your comment. It seems "OS test" TA hasn't been included into our really product. Can we keep it in ``stats`` PTA?

Comment (jforissier) [reply host/xtest/stats.c:332]: Please ignore my suggestion, we'll keep the thing in the stats PTA.

**Review by jforissier — COMMENTED** [review]
A couple of suggestions below, then:

`Reviewed-by: Jerome Forissier <jerome.forissier@linaro.org>`

File: host/xtest/stats.c [hunk 2]
```diff
@@ -312,6 +313,36 @@ static int stat_loaded_ta(int argc, char *argv[])
...
+	if (res != TEEC_SUCCESS)
+		errx(EXIT_FAILURE,
+		     "TEEC_InvokeCommand: res %#"PRIx32" err_orig %#"PRIx32,
+		     res, eo);
+
+	printf("REE time: %"PRId32" Seconds, %"PRId32" Milliseconds\n",
+			op.params[0].value.a, op.params[0].value.b);
+	printf("TEE time: %"PRId32" Seconds, %"PRId32" Milliseconds\n",
```
[the remark this answers is not in the document]
Comment (weizhaojiang) [reply host/xtest/stats.c]: done.

File: host/xtest/stats.c [hunk 3]
```diff
@@ -28,6 +28,7 @@
 #define STATS_CMD_ALLOC_STATS	1
 #define STATS_CMD_MEMLEAK_STATS	2
 #define STATS_CMD_TA_STATS	3
+#define STATS_CMD_SYSTEM_TIME	4
```
[the remark this answers is not in the document]
Comment (weizhaojiang) [reply host/xtest/stats.c]: sure, will do.

**Pull request merged**